### DISCUSSION TITLE
Failed Uploads

### DISCUSSION THREAD

**Discussion by Gubna-Tech** [community/community#111376] [Repositories]
### Select Topic Area

Bug

### Body

24+ hours since users started reporting uploads failing, any update to this?

There are several posts, but no additional information or help since this started happening roughly 30 hours ago.

![image](https://github.com/community/community/assets/75906618/7984a673-adc0-4a76-b96c-4824aa11bf52)

**Comment by ettaboyle** [comment]
**Community Update:** This post was locked because it was closed as a duplicate.
